src/linux.rs

```diff
@@ -136,6 +137,22 @@ impl Command {
         self
     }
 
+    /// Reassociate child process with a namespace specified by a file
+    /// descriptor
+    ///
+    /// `fd` argument is a file descriptor referring to a namespace
+    ///
+    /// 'ns' is an optional namespace type. `None` allows any type of
+    /// namespace
+    ///
+    /// See `man 2 setns` for further details
+    pub fn setns(&mut self, fd: RawFd, ns: Option<Namespace>)
```

Comment:
Usually, we decipher the names. I.e this one should be `set_namespace` or `attach_namespace` or `associate_with_namespace`